src/components/templates/HackathonForm.js

```diff
@@ -98,6 +102,13 @@ const HackathonForm = ({ initialState }) => {
     participation_type: Yup.string().required(
       "Participation type is required."
     ),
+    prize: Yup.string()
+      .min(10, "Prize must be at least 10 characters long.")
+      .max(50, "Prize cannot be more than 50 characters long.")
+      .required("Prize is required."),
```

Comment:
Hey, do you think 50 characters is going to be enough to explain a prize? I get if it was like $1,000 prize, but just thinking if people wanted to offer alternatives... tech equipment and such? Just a though...

Reply:
I am going to make it 100 characters.